Create your first meshProject.md

```diff
@@ -5,22 +5,26 @@ In order to make the guide nicer for the users replace the general screenshot wi
+ 3. First you have to give your project a name. The best practice is to use the name of the application, product, or project plus the environment type. The identifier is automatically generated but can be modified by you. We recommend to keep the identifier as it is. ![General Information](./assets/project-creation/choose-project-name.png "Naming")
+ 4. The lower part of the first page provides you the possibility to tag your meshProject. Some tags are required to be specified by you. You can recognize the required tags by the red line on the left side of the input field. Some of the tags you chose might have an impact on the landing zone you are allowed to chose if a policy exists that might restricts the usage. Click next:arrow_right:![Tags](./assets/project-creation/choose-tags.png "Tags")
  5. Select the payment method provided via the meshCustomer. Click next:arrow_right:
- 6. Now we come to the most important part. Choose at least one of the platforms for your future meshProject. The marketplace with interesting services provided by your organization is selected by default. If you want to have for example an AWS account pick AWS and select a provided meshLZ as well. Click next :arrow_right: ![Platform](https://github.com/meshcloud/User-Guides/blob/b8e2b6e183c82c86bc3e1b14173d267f5652281c/assets/PlatformAndLZ.png "Platform and Landing Zone")
- 7. Now add any team members who should have access to the new meshProject and the corresponding cloud tenants. You can also add yourself by clicking the "Add myself" button. Don't forget to confirm your action by clicking the "+" and selecting a role. ![Add Users and Groups](https://github.com/meshcloud/User-Guides/blob/cfbe2ccfd4fefed40e1468db2fc51e1ecda83cc9/assets/AddUsers.png "Add Users and Groups")
- 8. Finally! The moment has come and you can create your project. Click the "Create Project" button to confirm your new project. It will take a while to create the project and add the users to the tenants. Check the replication status of the new OpenShift Tenant. After a while it should look like this... ![Check Replication Status](https://github.com/meshcloud/User-Guides/blob/b8e2b6e183c82c86bc3e1b14173d267f5652281c/assets/ReplicationStatus.png "Add Users and Groups")
+ 6. Now we come to the most important part. Choose at least one of the platforms for your future meshProject. The marketplace with interesting services provided by your organization is selected by default. If you want to have for example an AWS account pick AWS and select a provided meshLZ as well. Click next :arrow_right: ![Platform and Landing Zone](./assets/project-creation/choose-platform-lz.png "Platform and Landing Zone")
+ 7. Now add any team members who should have access to the new meshProject and the corresponding cloud tenants. You can also add yourself by clicking the "Add myself" button. ![Add Users and Groups](./assets/project-creation/add-yourself.png "Add Users and Groups")
```

Comment:
would point out that it is possible to add team members also at a later point in time

Reply:
adapt it